packages/core/core-flows/src/order/workflows/return/confirm-return-request.ts

```diff
@@ -111,6 +99,31 @@ function prepareFulfillmentData({
   }
 }
 
+function extractReturnShippingOptionId({ orderPreview, orderReturn }) {
+  if (!orderPreview.shipping_methods?.length) {
+    return
+  }
+
+  let returnShippingMethod
+  for (const shippingMethod of orderPreview.shipping_methods) {
+    const modifiedShippingMethod_ = shippingMethod as any
+    if (!modifiedShippingMethod_.actions) {
+      continue
+    }
+
+    for (const action of modifiedShippingMethod_.actions) {
+      if (
+        action.action === ChangeActionType.SHIPPING_ADD &&
+        action.return_id === orderReturn.id
+      ) {
+        returnShippingMethod = shippingMethod
+        break
+      }
+    }
```

Comment:
question: If there is only one, we can just use `.find()` on the actions. Think that's a bit cleaner no?